.devcontainer/nginx/pmm.conf

```diff
@@ -0,0 +1,216 @@
+  upstream managed-grpc {
```

Comment:
same as in the dev container, the only difference is disabled auth
`auth_request off;`